src/components/DirectoryOrgs.tsx

```diff
@@ -33,9 +33,7 @@ const OrgDirectoryCards: FC<Props> = async ({ club }) => {
             <div className="h-7 w-7 text-white">
               <GroupIcon />
             </div>
-            <div className="ml-1 h-fit w-fit text-xs font-bold text-white">
-              {30} Members
-            </div>
+            
           </div>
```

Comment:
For this, we're good to also remove the group icon

Reply:
Yes I removed it and checked in the code